test/battle/item_effect/escape.c

```diff
@@ -0,0 +1,47 @@
+#include "global.h"
+#include "test/battle.h"
+
+WILD_BATTLE_TEST("Poke Toy lets the player escape from a wild battle")
+{
+    GIVEN {
+        ASSUME(gItems[ITEM_POKE_TOY].battleUsage == EFFECT_ITEM_ESCAPE);
```

Comment:
Can you replace all of the tests assumes with one at the top?

```
ASSUMPTIONS
{
    ASSUME(gItems[ITEM_POKE_TOY].battleUsage == EFFECT_ITEM_ESCAPE);
}
```

Reply:
Done.